rtgui/guiutils.cc

```diff
@@ -917,6 +917,11 @@ bool MyExpander::on_enabled_change(GdkEventButton* event)
     return false;
 }
 
+void MyExpander::setSensitive(bool makeSensitive)
```

Comment:
@Beep6581 I don't quite understand this one. Isn't a `MyExpander` a `Gtk::VBox`, so that you can call `set_sensitive()` directly?

Reply:
@Floessie you're right, it was unnecessary as-is.

Reply:
Cleaned up in latest commit.